book/doctrine.rst

```diff
@@ -519,11 +519,10 @@ of the bundle::
 .. tip::
 
     This article shows working with Doctrine from within a controller by using
-    the :method:`Symfony\\Bundle\\FrameworkBundle\\Controller\\Controller::getDoctrine`
-    method of the controller. This method is a shortcut to get the
-    ``doctrine`` service. You can work with Doctrine anywhere else
-    by injecting that service in the service. See
-    :doc:`/book/service_container` for more on creating your own services.
+    the ``getDoctrine()`` method of the controller. This method is a shortcut
+    to get the ``doctrine`` service. You can work with Doctrine anywhere else
+    by injecting that service in the service. See :doc:`/book/service_container`
```

Comment:
This should be reworded to improve its readability: `by injecting that service in the service`